tensorflow/lite/micro/kernels/pooling.h

```diff
@@ -71,14 +71,28 @@ void MaxPoolingEvalQuantized(TfLiteContext* context, TfLiteNode* node,
 TfLiteRegistration Register_AVERAGE_POOL_2D_INT8();
 
 TfLiteRegistration Register_MAX_POOL_2D_INT8();
+
+TfLiteRegistration Register_AVERAGE_POOL_2D_INT16();
```

Comment:
Very minor comment:

Let's group by the kernel instead of the quantization.

So have max_pool int8 and int16 next to each other.

And then average_pool int8 and int16 in a separate block of code.

Reply:
Good point, done